src/DataLayer/converters/tests/fixtures/VisObj.fixtures.ts

```diff
@@ -947,6 +947,66 @@ const oneMeasureOneAttributeWithIdentifiers: VisualizationObject.IVisualizationO
     filters: []
 };
 
+const twoAttributesAndNativeSubtotals: VisualizationObject.IVisualizationObjectContent = {
```

Comment:
also consider a use case with visualization object for pivot table
which have attributes in rows and columns 

Something like this...
```
{
   "visualizationObject":{
      "meta":{
         "title":"PivotTableExample",
         "category":"visualizationObject"
      },
      "content":{
         "buckets":[
            {
               "localIdentifier":"measures",
               "items":[
                  {
                     "measure":{
                        "localIdentifier":"0a79753c38754934841e0d5ca65d1239",
                        "definition":{
                           "measureDefinition":{
                              "item":{
                                 "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/9211"
                              }
                           }
                        },
                        "title":"_Close [BOP]"
                     }
                  },
                  {
                     "measure":{
                        "localIdentifier":"a223b7e9ac524b9a9ce13dd8b7a91448",
                        "definition":{
                           "measureDefinition":{
                              "item":{
                                 "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/9203"
                              }
                           }
                        },
                        "title":"_Close [EOP]"
                     }
                  }
               ]
            },
            {
               "localIdentifier":"attribute",
               "items":[
                  {
                     "visualizationAttribute":{
                        "localIdentifier":"f25253a742de4b6cb9bc903f2a55832b",
                        "displayForm":{
                           "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/324"
                        }
                     }
                  }
               ]
            },
            {
               "localIdentifier":"columns",
               "items":[
                  {
                     "visualizationAttribute":{
                        "localIdentifier":"2a6a74b43cfa40f68cf1b57dcbe503b5",
                        "displayForm":{
                           "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/1027"
                        }
                     }
                  },
                  {
                     "visualizationAttribute":{
                        "localIdentifier":"f6668412c866456cb1ba3e079ae28ee0",
                        "displayForm":{
                           "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/952"
                        }
                     }
                  }
               ]
            }
         ],
         "filters":[
            {
               "relativeDateFilter":{
                  "dataSet":{
                     "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/330"
                  },
                  "granularity":"GDC.time.year"
               }
            },
            {
               "negativeAttributeFilter":{
                  "displayForm":{
                     "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/1027"
                  },
                  "notIn":[

                  ]
               }
            },
            {
               "negativeAttributeFilter":{
                  "displayForm":{
                     "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/952"
                  },
                  "notIn":[

                  ]
               }
            }
         ],
         "visualizationClass":{
            "uri":"/gdc/md/o26s3o28sfgmv25pj76bdqtybf9wvuey/obj/75537"
         },
         "properties":"{\"sortItems\":[{\"attributeSortItem\":{\"attributeIdentifier\":\"f25253a742de4b6cb9bc903f2a55832b\",\"direction\":\"asc\"}}]}"
      }
   }
}
```

Reply:
done